package.json

```diff
@@ -28,7 +28,7 @@
     "lint": "npx eslint . --ext .ts",
     "lint:fix": "npx eslint . --ext .ts --fix",
     "test": "npx mocha -r ts-node/register test/**/*.spec.ts",
-    "build": "npx tsc",
+    "build": "rm -rf dist && npx tsc -p tsconfig-build.json",
```

Comment:
Perhaps we could create a `clean` script as well, such that:

```json
"clean": "rm -rf dist",
"build": "npx clean && npx tsc -p tsconfig-build.json"
```

This way we can have a logical place to insert these conditions?

Reply:
- [x] done